### DISCUSSION TITLE
Any way to support lisp language

### DISCUSSION THREAD

**Discussion by cxwx** [Wansmer/treesj#171] [Ideas]
As lisp use a lot of `()`. It there any way to support lisp languages

**Comment by Wansmer** [comment]
I don't see why it couldn't be handled with `treesj`. Personally, I don't write in any of the lisp dialects, so if you want it added to `treesj`, provide `joined/splitted` code samples similar to the examples in [tests/sample](https://github.com/Wansmer/treesj/tree/main/tests/sample). It would be even better if you create [PR](https://github.com/Wansmer/treesj/blob/main/tests/README.md) yourself.
